docs/ui/Rectangle.tsx

```diff
@@ -2,13 +2,18 @@ import type { ReactNode } from 'react';
 
 export interface RectangleProps {
   height?: string;
+  width?: string;
   children?: ReactNode;
 }
 
-export const Rectangle = ({ children, height }: RectangleProps) => (
+export const Rectangle = ({
+  children,
+  height,
+  width = '100%',
+}: RectangleProps) => (
   <div
-    className="w-full rounded-sm bg-gradient-to-r from-[hsl(29,_37%,_70%)] to-[hsl(29,_37%,_40%)] shadow"
-    style={{ height: height }}
+    className="rounded-sm border-2 border-dashed border-[hsl(29,_37%,_40%)] bg-[hsl(29,_37%,_70%)]"
```

Comment:
I think if we want it to look more like a placeholder, it should not be that far I'm when I did the rectangles, I made them brown because of the docs theme at this point. I think now it should be just gray?

Maybe be can use transparent with the "neutral" tailwind color. So it looks good in all themes?

Reply:
We can, I don't want to make a big difference because I think there was an intension :D

Reply:
have a look again